apps/examples/smartfs/fs_performance/fs_performance_main.c

```diff
@@ -0,0 +1,461 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <tinyara/timer.h>
+
+/****************************************************************************
+ * Pre Processor Definitions
+ ****************************************************************************/
+
+#define ROOT				"/mnt/"
+#define TEST_FILE			ROOT "test_file"
+#define TEST_FLAG_FILE_CREATION		1
+#define TEST_FLAG_OVERWRITE		2
```

Comment:
It's more readable because defining macro those constant integer. It seems good.